index.js

```diff
@@ -165,15 +183,25 @@ class ServerlessPlugin {
 
     serverless.cli.log(`Targeting ${serverless.service.__cljsArtifact}`);
 
-    destroyFnMap(serverless.service);
+    setCljsLambdaFnMap(serverless);
+    setCljsLambdaExclude(serverless);
 
     const buildAndMerge = after_createDeploymentArtifacts.bind(
       null, serverless, opts);
 
-    this.hooks = {
-      'after:deploy:createDeploymentArtifacts': buildAndMerge,
-      'after:deploy:function:packageFunction': buildAndMerge
-    };
+    if(!isLumo(serverless)) {
```

Comment:
This might be easier to follow as:

```js
const when = isLumo(serverless) ? 'before' : 'after';

this.hooks = {
  [`${when}:deploy:createDeploymentArtifacts`]: buildAndMerge,
  ...}
```

Reply:
nice ;)

Reply:
So do you want to standardize on `package.createDeploymentArtifacts` for this one? In any case it is called on `deploy`.